src/actions.js

```diff
@@ -41,6 +41,23 @@ export const setLayout = layout =>
     dispatch(setLayoutAction({ layout }));
   };
 
+export const SET_CURRENT_CONTENT = '@app/SET_CURRENT_CONTENT';
+export const REMOVE_CURRENT_CONTENT = '@app/REMOVE_CURRENT_CONTENT';
+
+export const setCurrentContent = content => (dispatch) => {
```

Comment:
We probably can use `createAction` from `redux-actions` for `setCurrentContent` and `removeCurrentContent`.